### ISSUE TITLE
No issues raised on heading inside of ARIA radio

### ISSUE THREAD

**Issue by anevins12** [validator/validator#1530]
Code being validated:
```
<div role="radio" aria-checked="true">
<h2> A heading </h2>
</div>
```
Results with:
> Document checking completed. No errors or warnings to show.

**Comment by pkra** [comment]
While I'd call this bad, I don't see how any ARIA spec forbids this.

**Comment by sideshowbarker** [comment]
Closed per https://github.com/validator/validator/issues/1530#issuecomment-3473519218

**Comment by stevefaulkner** [comment]
there is the advice in ARIA in HTML on allowed descendants of roles https://www.w3.org/TR/html-aria/#allowed-descendants-of-aria-roles but that is currently not normative
